python/api/src/araali/quickstart_bend.py

```diff
@@ -0,0 +1,47 @@
+from . import utils
+import requests
+import yaml
+
+MAX_RETRIES = 5
+
+def create_tenant_and_token(email):
+    host = "https://console-%s.aws.araalinetworks.com" % utils.cfg["backend"]
+    # Make request to quickstart backend to create tenant and fetch workloadYaml
+    for x in range(MAX_RETRIES):
```

Comment:
Just added re-tries to avoid random failures, but didn't encounter any failures. It goes through first time every attempt, but yeah just in case backend is down for 30 seconds or something or some flakiness.